src/STW.StubApi.Presentation/Services/CommodityCode/Controllers/CommodityCategoryController.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.AspNetCore.Mvc;
+using STW.StubApi.Presentation.Services.CommodityCode.Helpers;
+
+namespace STW.StubApi.Presentation.Services.CommodityCode.Controllers;
+
+[ApiController]
+[Route("commodity-categories")]
+public class CommodityCategoryController : ControllerBase
+{
+    [HttpGet("{certType}-{commodityCode}")]
+    public IActionResult GetCommodityCategory([FromRoute] string certType, [FromRoute] string commodityCode)
+    {
+        return Ok(CommodityCategoriesHelper.GetCommodityCategory(certType, commodityCode));
```

Comment:
If you're always returning a successful response you can just set the return type to be the object returned and drop `Ok()`